src/SqlBinding/SqlAsyncCollector.cs

```diff
@@ -340,15 +380,15 @@ public async static Task<TableInformation> RetrieveTableInformationAsync(SqlConn
                 catch (Exception ex)
                 {
                     // Throw a custom error so that it's easier to decipher.
-                    string message = $"Encountered exception while retrieving column names and types for table '{tableName}' in schema '{schema}.' Cannot generate upsert command without them.";
+                    string message = $"Encountered exception while retrieving column names and types for table '{tableName}' in schema {schema}. Cannot generate upsert command without them.";
```

Comment:
Removing the extra quotes around schema since GetTableAndSchema returns schema with quotes around it.

Reply:
If the schema is always returned with quotes around it, then at the very least we should include that in the comments above the `GetTableAndSchema` method.

However, I think we should have consistency here. Either both table name + schema should have quotes around them, or neither should from `GetTableAndSchema`. Let's just be consistent here.

Reply:
I think this is because schema can also be USER_SCHEMA(), which is returned without quotes. For consistency, we can return both tableName and schema with quotes.